Test/function_test.py

```diff
@@ -318,6 +318,12 @@
 'None'
 >>> ERR_calc(0.1)
 0.9
+>>> cm.average("PPV")
```

Comment:
@alirezazolanvari Would you please change this test case (use another parameter)?
It seems there is a different in averaging method in `Python 3.5`. 
```
Expected :
0.5666666666666668
Got : 
0.5666666666666667
```

Reply:
@sepandhaghighi you can keep on now ;)